### ISSUE TITLE
Unsupported URL

### ISSUE THREAD

**Issue by h-q** [rg3/youtube-dl#13959]
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2017.08.18**
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Site support request (request for adding support for a new site)

Dear youtube-dl. I appreciate adding support (if possible) to the following sites:

- Single video: http://ara.tv/gbetq
- Single video: http://www.iranproud.online/iran-movies/drama/adat-nemikonim?ghiXbed2Vi8MrhmpgJYi

Thanks a million!

**Comment by kayb94** [comment]
I can not load the first one at all. The second one looks like not respecting copyright, which is against youtube-dl's policy as far as I know. So there probably won't be no support. Sorry for that.

**Comment by h-q** [comment]
Thank you @kayb94 for your response. For the first URL, are you able to load:
http://www.alarabiya.net/ar/khota/season2/2016/05/06/%D8%A7%D9%84%D8%AD%D9%84%D9%82%D8%A9-30.html
? I wonder if it's region-restricted (shouldn't be though). It's a public news site.

And thanks for informing me about the 2nd URL.

**Comment by kayb94** [comment]
Yes, I can acces that link you provided. Might have been because of my NoScript browser plugin or so. Maybe I'll add support next week, now I don't have time (no promise though). It looks possible.

Regards

**Comment by h-q** [comment]
Thank you @kayb94. You're awesome!
